docs/sources/developers/plugins/create-a-grafana-plugin/develop-a-plugin/build-a-logs-data-source-plugin.md

```diff
@@ -197,6 +197,34 @@ const result = new MutableDataFrame({
 });
 ```
 
+### Logs to trace using data link with url
+
+If your log data contains **trace IDs**, you can enhance your log data frames by adding a **field with trace ID values** and **URL data links**. These links should utilize the trace ID value to accurately link to the appropriate trace. This enhancement enables users to seamlessly move from log lines to the relevant traces.
+
+**Example in TypeScript:**
+
+```ts
+const result = new MutableDataFrame({
```

Comment:
`MutableDataFrame` is deprecated. WDYT about just using `DataFrame`?

```suggestion
const result = new DataFrame({
```

Reply:
Good point!  I discovered `createDataFrame` function that will create `dataFrame` and used it in https://github.com/grafana/grafana/pull/73225/commits/d88f23bbd1fba3dfa316b55ef83b9af7c647b610. Other option would be just using `result: DataFrame = {...}`, but then we would need to ensure that `length` is everywhere as it is required field. This handles it for us. Let me know what do you think.

Reply:
I like that! 👍